python/datasources/vera_incarceration_county.py

```diff
@@ -36,6 +43,11 @@
     POP: "incarceration_population_pct"
 }
 
+PCT_REL_INEQUITY_COL_MAP = {
+    JAIL: "jail_pct_relative_inequity",
```

Comment:
Just curious, should these be constants, or is this the better pattern for this data source?

Reply:
good call, that is the current pattern. Though I'm wondering now why we declare variable-specific constants globally.... seems like the only stuff that should be in that std_col file are things used across the codebase?